librz/core/cmd/cmd_help.c

```diff
@@ -430,6 +430,102 @@ RZ_IPI void rz_core_clippy_print(RzCore *core, const char *msg) {
 	}
 }
 
+RZ_API void rz_core_cmd_help_calc_expr(RzCore* data, const char* input) {
+	RzCore* core = (RzCore*)data;
```

Comment:
why you need to cast something that is `RzCore` already into `RzCore` ? just rename `data` argument into `core` and delete this line.

Reply:
I'm sorry, this was a side effect of copy pasting and forgot to change that. Will be changed in upcoming commit.